### PR TITLE
WebDriver conformance changes for Firefox 91

### PR DESCRIPTION
**Pull request by juliandescottes**
The following PR contains all the user facing changes to Marionette for the Firefox 91 release. Here a [list of all the changes](https://bugzilla.mozilla.org/buglist.cgi?j_top=OR&f1=cf_status_firefox91&o1=equals&resolution=FIXED&o2=equals&query_format=advanced&f2=cf_status_firefox91&v1=fixed&component=Marionette&v2=verified&product=Testing&).

@whimboo can you please cross-check? Thanks.

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/91/index.html [hunk 1]
```diff
@@ -80,8 +80,11 @@ <h3 id="WebAssembly">WebAssembly</h3>
 <h4 id="removals_wasm">Removals</h4>
 
 <h3 id="webdriver_conformance_marionette">WebDriver conformance (Marionette)</h3>
-
-<h4 id="removals_webdriver">Removals</h4>
+<ul>
+  <li>Fixed a bug which prevented to close prompts opened by popup windows, which could make Marionette hang ({{bug(1721982)}}).</li>
```
Comment (whimboo) [inline files/en-us/mozilla/firefox/releases/91/index.html:84]:
```suggestion
  <li>Fixed a bug, which caused the commands <code>WebDriver:AcceptAlert</code> and <code>WebDriver:DismissAlert</code> to hang for user prompts as opened by a popup window ({{bug(1721982)}}).</li>
```

File: files/en-us/mozilla/firefox/releases/91/index.html [hunk 2]
```diff
@@ -80,8 +80,11 @@ <h3 id="WebAssembly">WebAssembly</h3>
...
+  <li>Fixed a inappropriate handling of the <code>webSocketUrl</code> capability, which would return "true" if <code>webSocketUrl</code> was not supported ({{bug(1713775)}}).</li>
+  <li>Minor performance improvement related to unnecessary log operations ({{bug(1708532)}})</li>
```
Comment (whimboo) [inline files/en-us/mozilla/firefox/releases/91/index.html:86]: I though a bit more about that entry, and maybe it's not worth mentioning it for the release notes.

**Review by whimboo — CHANGES_REQUESTED** [review]
Some small changes that I would propose. Let me know what you think.

**Comment by juliandescottes** [comment]
Thanks, applied suggestions.

File: files/en-us/mozilla/firefox/releases/91/index.html [hunk 3]
```diff
@@ -80,8 +80,10 @@ <h3 id="WebAssembly">WebAssembly</h3>
 <h4 id="removals_wasm">Removals</h4>
 
 <h3 id="webdriver_conformance_marionette">WebDriver conformance (Marionette)</h3>
-
-<h4 id="removals_webdriver">Removals</h4>
+<ul>
+  <li>Fixed a bug, which caused the commands <code>WebDriver:AcceptAlert</code> and <code>WebDriver:DismissAlert</code> to hang for user prompts as opened by a popup window ({{bug(1721982)}}).</li>
```
Comment (whimboo) [inline files/en-us/mozilla/firefox/releases/91/index.html:84]:
```suggestion
  <li>Fixed a bug, which caused the commands <code>WebDriver:AcceptAlert</code> and <code>WebDriver:DismissAlert</code> to hang for user prompts as opened in a popup window ({{bug(1721982)}}).</li>
```

Comment (whimboo) [reply files/en-us/mozilla/firefox/releases/91/index.html:84]: Sorry, I missed that part. It's not that the popup actually opens the user prompt.

**Review by whimboo — APPROVED** [review]
r=me with the nit fixed.

**Comment by whimboo** [comment]
@Rumyra could you please have a look? Thanks.

**Review by Rumyra — APPROVED** [review]

**Comment by Rumyra** [comment]
This looks great to me - I've approved - can someone confirm the above conversations are resolved - thank you!

Let me know if there's anything else, I'm currently working on a personal branch tidying up the release notes, so I can add anything there 👍

**Comment by juliandescottes** [comment]
> This looks great to me - I've approved - can someone confirm the above conversations are resolved - thank you!
> 
> Let me know if there's anything else, I'm currently working on a personal branch tidying up the release notes, so I can add anything there 👍

Thanks!

All the initial review comments were folded in the first commit, and the additional nit was automatically added as a new commit. Everything should be resolved.

**Comment by Rumyra** [comment]
Thanks @juliandescottes 👍

**Pull request merged**